truffle.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
     }
   },
   mocha: {
-    reporter: "mocha-circleci-reporter",
+    reporter: "eth-gas-reporter",
```

Comment:
Would be nice if there was an easy way to exclude this change from the PR (since it's an artifact of our test setup)

Reply:
That's my mistake for committing it. Will undo.